### ISSUE TITLE
ui: Scrollbar for the chat conversations goes behind the Message Box

### ISSUE THREAD

**Issue by LombudXa** [open-webui/open-webui#2504]
# Bug Report

## Description
The scrollbar on the right side disappears behind the footer of the conversation or in other words, behind the message box. Nothing very important but it gets a bit annoying when you want to use the mouse to scroll when having long conversations. So, I have to use the mouse wheel first.

**Bug Summary:**
Scrollbar behind message box.

**Steps to Reproduce:**
Just when you have a conversation with some content, you should notice immediately. There's no scrollbar to the right side to scroll through the conversation. So, you have to use your mouse wheel first to scroll up until it appears again.

**Expected Behavior:**
Scrollbar should stop before the message box.

**Actual Behavior:**
Scrollbar disappears behind message box.

## Environment

- **Open WebUI Version:** [0.1.125] - 2024-05-19
- **Ollama (if applicable):** v0.1.38

- **Operating System:** Windows 11 23H2
- **Browser (if applicable):** Happens with Firefox, Edge and Chrome (didn't test with others)

## Reproduction Details
See above or screenshots.

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.

## Logs and Screenshots

**Screenshots (if applicable):**
Scrollbar fully visible (on the right side):
![Scrollbar_fully_visible_01](https://github.com/open-webui/open-webui/assets/13348327/fdf4cf9c-5a84-48da-99cd-f74c72c51328)

Scrollbar starting to disappear when scrolling down (on the right side):
![Scrollbar_disappearing_02](https://github.com/open-webui/open-webui/assets/13348327/47b48a65-5b4e-4044-ad81-6046091bccaf)

Scrollbar gone when at the end of the conversation (on the right side):
![Scrollbar_gone_03](https://github.com/open-webui/open-webui/assets/13348327/476f8a12-7d50-49ac-98ba-60c32716bae3)

## Installation Method

Docker (Debian system)

## Additional Information

Not aware of anything to add here.

**Comment by gamepad-coder** [comment]
Possibly related to:
- https://github.com/open-webui/open-webui/issues/2231

Patching that one might resolve this one.

**Comment by gamepad-coder** [comment]
Note from other thread: #2231 was closed as a duplicate of this issue.

**Comment by gamepad-coder** [comment]
Consider also testing the use case for [#2504 ui: Scrollbar for the chat conversations goes behind the Message Box](https://github.com/open-webui/open-webui/issues/2504) when you do the implementation for this #2504 patch.

Because Firefox on Android is totally unable to view the buttons under the most recent Response right now.

**Comment by tjbck** [comment]
No need to spam, it will be fixed 😅

**Comment by gamepad-coder** [comment]
Apologies, didn't mean to pressure / didn't mean to convey a sense of urgency.
I just like linking issues so it's easy to see the history / cross reference.

Habit from work :)

**Comment by LombudXa** [comment]
Yeah sorry, I have read that issue in fact but I was not sure it's actually the same as what I've described. Also, the scroll bar should not stop before the message box, it probably should just be on top of it.

Anyway, thanks for fixing it.

**Comment by tjbck** [comment]
Fixed on dev! Let us know if the issue persists!

**Comment by LombudXa** [comment]
Fixed :) Looks good now. Thanks.

**Comment by gamepad-coder** [comment]
@tjbck Wow great job! 🎉 

Confirmed both issues (this https://github.com/open-webui/open-webui/issues/2504 #2587) and are fully patched in all the following browsers
- desktop chrome
- desktop firefox
- android chrome
- android firefox

Tested with default zoom, increased zoom, and increased input box size with a bunch of newlines.

----

PS: I just skimmed through all the commits in the v0.2.0 release! It's soo neat to stumble onto a repo that's being actively maintained and extended. This project is really really cool! Keep it up!
